recipes/python-irodsclient/LICENSE.txt

```diff
@@ -0,0 +1,14 @@
+Copyright (c) 2014-2017, The University of North Carolina at Chapel Hill
```

Comment:
You can remove this license file since it's packaged with the source distribution.